docs/examples/envbinding/deploy-and-bind-cloud-resource.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: rds-app
+  namespace: project-1
+spec:
+  components:
+    - name: db
+      type: alibaba-rds
+      properties:
+        instance_name: db
+        account_name: kubevela
+        password: my-password
+        writeConnectionSecretToRef:
+          name: project-1-rds-conn-credential
+  policies:
+    - name: env-policy
+      type: env-binding
+      properties:
+        envs:
+          # 部署 RDS 给杭州集群
+          - name: hangzhou
+            placement:
+              clusterSelector:
+                name: cluster-hangzhou
+            patch:
+              components:
+                - name: db
+                  type: alibaba-rds
+                  properties:
+                    # region: hangzhou
+                    instance_name: hangzhou_db
+          # 部署 RDS 给香港集群
+          - name: hongkong
+            placement:
+              clusterSelector:
+                name: cluster-hongkong
+              namespaceSelector:
+                name: hk-project-1
+            patch:
+              components:
+                - name: db
+                  type: alibaba-rds
+                  properties:
+                    # region: hongkong
+                    instance_name: hongkong_db
+                    writeConnectionSecretToRef:
+                      name: hk-project-rds-credential
+
+  workflow:
+    steps:
+      # 部署 RDS 给杭州区用
+      - name: deploy-hangzhou-rds
+        type: deploy-cloud-resource
```

Comment:
In our last discussion, `deploy-cloud-resource` should not exist. Without specifying a cluster, `deploy2env` means deploying the workload in the managed cluster. Right?

Reply:
Right. But that is **deploy2env**, not **deploy-cloud-resource**, they are different things. **deploy-cloud-resource** only deploy terraform configuration in manage cluster. Only the secret will be dispatched to child clusters.

Reply:
If you want to dispatch terraform configuration into child clusters, you can switch **deploy-cloud-resource** to **deploy2env**.

Reply:
![image](https://user-images.githubusercontent.com/2805315/142554348-38d33774-f9d5-478a-b13c-d4be44f42a0e.png)

It would be simple and clear to end-users if we invent fewer idioms. In this decision we made above, we can still use `deploy2env` without setting parameter `env`, which means not to dispatch Configuration to child clusters.

```  workflow:
    steps:
      - name: deploy-test-server
        type: deploy2env
        properties:
          policy: env
          env: test
```

A Terraform application is special, but we are aiming to treat it as a normal application as possible.

Reply:
`deploy-cloud-resource` is correct for me. They have different logic. `deploy-cloud-resource` only deploy the secret into child cluster while `deploy2env` will deploy the whole cloud resource object.